addons/website_links/static/src/js/website_links.js

```diff
@@ -425,7 +425,7 @@ publicWidget.registry.websiteLinks = publicWidget.Widget.extend({
      * @private
      */
     _onShortenUrlButtonClick: async function (ev) {
-        const textValue = ev.target.dataset["clipboard-text"];
+        const textValue = ev.target.dataset["clipboard-text"] || ev.target.dataset["clipboardText"];
```

Comment:
I think "clipboard-text" could be removed?

side note: it's not too clear to me from commit message, but I think dataset are automatically camelCased (https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset#name_conversion) so the fix just makes sense